src/components/Heading/Heading.tsx

```diff
@@ -10,7 +11,9 @@ export type Props = {
   children: ReactNode
   /** テキストのスタイル */
   type?: HeadingTypes
-  /** コンポーネントの HTML タグ */
+  /**
+   * @deprecated tag属性は非推奨です。SectioningContent(Article, Aside, Nav, Section, SectioningFragment)を使ってHeadingと関連する範囲を明確に指定してください
```

Comment:
nits: `@deprecated` 付けてるので、対処方法だけ案内でよさそう

```suggestion
   * @deprecated SectioningContent(Article, Aside, Nav, Section, SectioningFragment)を使ってHeadingと関連する範囲を明確に指定してください
```